src/object/obj_verify.c

```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright 2019-2021 Intel Corporation.
+ * (C) Copyright 2019-2022 Intel Corporation.
```

Comment:
Copyright change probably happened when a merge tripped the commit hook.

Reply:
undone